lawnchair/res/values-pl-rPL/strings.xml

```diff
@@ -160,6 +160,7 @@
     <string name="icon_shape_egg">Jajko</string>
     <string name="icon_shape_cupertino">iOS</string>
     <string name="icon_shape_octagon">Ośmiokątny</string>
+    <string name="icon_shape_hexagon">Sześciokątny</string>
```

Comment:
Lawnchair Launcher only uploads the source text (strings.xml) to Crowdin so this change is likely going to be reverted in the next translation sync. 

I'd recommend translating this string once this PR has been merged (and Crowdin Upload CI has already done its job.)

Reply:
Sure, I removed the translation from PR.